# translation of uk(5).po to Ukrainian
# Maxim Dziumanenko <mvd@mylinux.ua>, 2005.
# Vadim Abramchuck <Abram@email.ua>, 2006.
# Ukrainian translation of update-manager.
# Copyright (C) 2005, 2006 Free Software Foundation, Inc.
msgid ""
msgstr ""
"Project-Id-Version: uk(5)\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2024-03-14 18:28+0000\n"
"PO-Revision-Date: 2023-02-21 12:28+0000\n"
"Last-Translator: Volodymyr Shypovych <Unknown>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2024-03-25 09:28+0000\n"
"X-Generator: Launchpad (build aec24aef7a9042c99ef3e238d8b0ca01df9e1a9f)\n"
"Language: uk\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:474
#, python-format
msgid "Server for %s"
msgstr "Сервер для %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:225 ../DistUpgrade/distro.py:237
#: ../DistUpgrade/distro.py:258
msgid "Main server"
msgstr "Основний сервер"

#: ../DistUpgrade/distro.py:267
msgid "Custom servers"
msgstr "Сервери користувача"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Вилучити пакунок, що знаходиться в поганому стані"
msgstr[1] "Вилучити пакунки, що знаходяться в поганому стані"
msgstr[2] "Вилучити пакунки, що знаходяться в поганому стані"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Пакунок '%s' знаходиться у несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Чи ви бажаєте "
"видалити цей пакунок зараз, щоб продовжити?"
msgstr[1] ""
"Пакунки'%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"
msgstr[2] ""
"Пакунки '%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "Схоже, що сервер перенавантажений"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Пошкоджені пакунки"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Ваша система містить пошкоджені пакунки, котрі не можуть бути виправлені "
"цією програмою. Будь-ласка, виправіть їх програмами synaptic або apt-get."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Пошкоджені пакети після оновлення: %s"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:665
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""
"Під час розрахунку оновлення виникла нерозв'язувана проблема.\n"
"\n"
" "

#: ../DistUpgrade/DistUpgradeCache.py:668
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""
"Ймовірно, причиною є:\n"
" * Неофіційні пакунки із програмним забезпеченням, які не є частиною Ubuntu\n"
"Будь ласка, скористайтеся програмою ppa-purge з пакунка ppa-purge \n"
"для вилучення програмного забезпечення з PPA Launchpad, а потім \n"
"повторіть спробу оновлення.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:676
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""
"Причина:\n"
" * Оновлення до тестової версії Ubuntu\n"
"Ймовірно, проблема є тимчасовою. \n"
"Будь ласка, повторіть спробу пізніше.\n"

#: ../DistUpgrade/DistUpgradeCache.py:683
msgid "This is most likely a transient problem, please try again later."
msgstr "Перелік змін ще не готовий, спробуйте пізніше."

#: ../DistUpgrade/DistUpgradeCache.py:686
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""
"Якщо жодне із цих припущень є не правильним, будь ласка, повідомте про цю "
"ваду за допомогою команди «ubuntu-bug ubuntu-release-upgrader-core», "
"віддавши її у терміналі. "

#: ../DistUpgrade/DistUpgradeCache.py:688
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at "
"'main.log' and 'apt.log'."
msgstr ""
"Якщо захочете розібратися із причинами власноруч, у файлах журналу у "
"/var/log/dist-upgrade міститимуться подробиці щодо оновлення. Особливу увагу "
"зверніть на main.log та apt.log."

#: ../DistUpgrade/DistUpgradeCache.py:694
msgid "Could not calculate the upgrade"
msgstr "Не вдалося розрахувати оновлення"

#: ../DistUpgrade/DistUpgradeCache.py:751
msgid "Error authenticating some packages"
msgstr "Помилка аутенфікації деяких пакетів"

#: ../DistUpgrade/DistUpgradeCache.py:752
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Не вдалося перевірити деякі пакети. Це може бути викликано проблемами в "
"мережі. Можливо, Ви захочете спробувати пізніше. Список не перевірених "
"пакетів надано нижче."

#: ../DistUpgrade/DistUpgradeCache.py:773
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal deny list."
msgstr ""
"Пакунок '%s' позначено для видалення, але він знаходиться в списку "
"заборонених на видалення."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Важливий пакунок '%s' помічено для видалення."

#: ../DistUpgrade/DistUpgradeCache.py:788
#, python-format
msgid "Trying to install deny listed version '%s'"
msgstr "Спроба встановити заборонену версію '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:910
#, python-format
msgid "Can't install '%s'"
msgstr "Не можливо встановити '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:911
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Не вдалося встановити потрібний пакунок. Повідомте про цю помилку, "
"використавши команду 'ubuntu-bug ubuntu-release-upgrader-core' в терміналі."

#: ../DistUpgrade/DistUpgradeCache.py:922
msgid "Can't guess meta-package"
msgstr "Не можливо підібрати meta-пакунок"

#: ../DistUpgrade/DistUpgradeCache.py:923
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Ваша система не містить такого %s чи такого %s пакунку, було неможливо "
"визначити, якою версією Убунту ви користуєтеся.\n"
" Будь ласка, встановіть один з пакунків вище за допомогою synaptic або apt-"
"get, перед тим як продовжити."

#: ../DistUpgrade/DistUpgradeController.py:145
msgid "Reading cache"
msgstr "Зчитування кешу"

#: ../DistUpgrade/DistUpgradeController.py:267
msgid "Unable to get exclusive lock"
msgstr "Не вдалося отримати ексклюзивне блокування"

#: ../DistUpgrade/DistUpgradeController.py:268
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Це зазвичай значить, що запущено інший менеджер пакунків (наприклад, apt-get "
"або aptitude). Будь ласка, спочатку закрийте цю програму."

#: ../DistUpgrade/DistUpgradeController.py:326
msgid "Upgrading over remote connection not supported"
msgstr "Віддалене оновлення системи більше не підтримується"

#: ../DistUpgrade/DistUpgradeController.py:327
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Ви намагаєтесь виконати оновлення через ssh-з'єднання з непідтримуваним "
"клієнтом. Оновіться у текстовому режимі через \"do-release-upgrade\".\n"
"\n"
"Оновлення зупинене. Спробуйте без ssh."

#: ../DistUpgrade/DistUpgradeController.py:341
msgid "Continue running under SSH?"
msgstr "Продовжити роботу через SSH?"

#: ../DistUpgrade/DistUpgradeController.py:342
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Ця сесія, скоріш за все, працює через ssh. Наполегливо рекомендуємо не "
"виконувати оновлення через ssh через те, що в разі невдачі буде важче "
"відновитись.\n"
"\n"
"Якщо Ви продовжите, буде запущено додаткову службу ssh на порті '%s'.\n"
"Чи бажаєте продовжити?"

#: ../DistUpgrade/DistUpgradeController.py:356
msgid "Starting additional sshd"
msgstr "Запуск додаткового sshd"

#: ../DistUpgrade/DistUpgradeController.py:357
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Щоб зробити відновлення легшим у разі поломки, на порті '%s' буде запущений "
"додатковий sshd. Якщо щось піде не так із запуском ssh, Ви все ще зможете "
"під'єднатись до другого.\n"

#: ../DistUpgrade/DistUpgradeController.py:365
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Якщо брандмауер увімкнений, треба тимчасово відкрити цей порт. Це потенційно "
"небезпечно, тому потребує втручання користувача. Ви можете відкрити порт "
"наступним чином:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:443
#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Can not upgrade"
msgstr "Оновлення неможливе"

#: ../DistUpgrade/DistUpgradeController.py:444
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Оновлення з '%s' до '%s' цією програмою не підтримується."

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""
"Інсталяція python3 install пошкоджена. Виправте символьне посилання "
"'/usr/bin/python3'."

#: ../DistUpgrade/DistUpgradeController.py:497
#, python-format
msgid "Can not write to '%s'"
msgstr "Неможливо виконати запис в '%s'"

#: ../DistUpgrade/DistUpgradeController.py:498
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Неможливо виконати запис в системний каталог '%s' у Вашій системі. Оновлення "
"не може бути продовженим.\n"
"Переконайтеся, що системний каталог доступний для запису."

#: ../DistUpgrade/DistUpgradeController.py:717
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "Не для людей, поки знаходиться на стадії розробки релізу %s"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid "disabled on upgrade to %s"
msgstr "відключено під час оновлення до %s"

#: ../DistUpgrade/DistUpgradeController.py:1218
msgid "No valid source entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1219
#: ../DistUpgrade/DistUpgradeController.py:1287
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Під час сканування даних вашого сховища неможливо знайти запис про %s.\n"
"\n"
"Оновлення може не відбутися.\n"
"\n"
"Продовжувати в будь-якому випадку?"

#: ../DistUpgrade/DistUpgradeController.py:1229
#: ../DistUpgrade/DistUpgradeController.py:1297
msgid "No valid mirror found"
msgstr "Не знайдено правильного дзеркала"

#: ../DistUpgrade/DistUpgradeController.py:1230
#: ../DistUpgrade/DistUpgradeController.py:1298
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"При скануванні сховища не було знайдено дзеркало для оновлення. Таке могло "
"статися у випадку використання внутрішнього дзеркала або інформація про "
"дзеркала застаріла.\n"
"\n"
"Все одно бажаєте переписати файл джерел 'sources.list'? Якщо ви вкажете "
"\"Так, тоді оновляться усі записи від '%s' до '%s'.\n"
"Якщо вкажете \"Ні\" оновлення буде припинено."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:1250
#: ../DistUpgrade/DistUpgradeController.py:1318
msgid "Generate default sources?"
msgstr "Створити типові джерела?"

#: ../DistUpgrade/DistUpgradeController.py:1251
#, python-format
msgid ""
"After scanning your 'ubuntu.sources' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1269
#: ../DistUpgrade/DistUpgradeController.py:1370
msgid "Third party sources disabled"
msgstr "Сторонні джерела відключені"

#: ../DistUpgrade/DistUpgradeController.py:1270
#: ../DistUpgrade/DistUpgradeController.py:1371
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Деякі сторонні джерела у Вашому sources.list були відключені. Ви можете "
"знову включити їх після оновлення за допомогою програми 'software-"
"properties' чи менеджера пакунків."

#: ../DistUpgrade/DistUpgradeController.py:1286
msgid "No valid sources.list entry found"
msgstr "Не знайдено жодних дійсних записів source.list"

#: ../DistUpgrade/DistUpgradeController.py:1319
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Після сканування вашого 'sources.list' не було знайдено дійсного запису для "
"'%s'.\n"
"\n"
"Додати типовий запис для '%s'? Якщо ви оберете 'Ні', оновлення буде "
"скасовано."

#: ../DistUpgrade/DistUpgradeController.py:1354
#: ../DistUpgrade/DistUpgradeController.py:1360
msgid "Repository information invalid"
msgstr "Помилка в даних про сховище"

#: ../DistUpgrade/DistUpgradeController.py:1355
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Оновлення інформації про репозиторій закінчилося невдачею (недійсний файл), "
"тому запустився процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:1361
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Спроба оновлення даних щодо сховища призвела до спотворення даних у файлі. "
"Щоб повідомити про ваду, встановіть apport, а потім віддайте команду «apport-"
"bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1413
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакунок у незадовільному стані"
msgstr[1] "Пакунки у незадовільному стані"
msgstr[2] "Пакунків у незадовільному стані"

#: ../DistUpgrade/DistUpgradeController.py:1416
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Пакунок '%s' знаходиться в несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Будь ласка, "
"перевстановіть пакунок вручну чи вилучіть його з системи."
msgstr[1] ""
"Пакунки '%s' знаходяться в несумісному стані, було б добре їх "
"перевстановити, та не вдається знайти для них ніяких архівів. Будь ласка, "
"перевстановіть пакунки вручну чи вилучіть їх із системи."
msgstr[2] ""
"Пакунки '%s' знаходяться в несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Будь ласка, перевстановіть пакунки "
"вручну чи вилучіть їх з системи."

#: ../DistUpgrade/DistUpgradeController.py:1477
msgid "Error during update"
msgstr "Помилка під час оновлення"

#: ../DistUpgrade/DistUpgradeController.py:1478
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Під час оновлення сталася помилка. Це зазвичай викликано проблемами у "
"мережі, будь ласка, перевірте Ваше мережеве з’єднання та спробуйте знову."

#: ../DistUpgrade/DistUpgradeController.py:1521
#: ../DistUpgrade/DistUpgradeController.py:1526
msgid "EFI System Partition (ESP) not usable"
msgstr "Системний розділ EFI (ESP) не підходить для використання"

#: ../DistUpgrade/DistUpgradeController.py:1522
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"Системний розділ EFI (ESP) не змонтований на /boot/efi. Переконайтеся, що "
"він належним чином налаштований і спробуйте ще раз."

#: ../DistUpgrade/DistUpgradeController.py:1527
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"Системний розділ EFI (ESP), що змонтований на /boot/efi недоступний для "
"запису. Змонтуйте його у режим читання-запису і спробуйте ще раз."

#: ../DistUpgrade/DistUpgradeController.py:1534
msgid "Not enough free disk space"
msgstr "Недостатньо місця на диску"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1537
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"Оновлення перервано. Для здійснення оновлення потрібно {str_total} вільного "
"місця на диску \"{str_dir}\". Будь ласка звільніть принаймні ще {str_needed} "
"дискового простору на \"{str_dir}\". {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1542
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Видаліть тимчасові пакунки попередніх інсталяцій,  виконавши команду 'sudo "
"apt clean'."

#: ../DistUpgrade/DistUpgradeController.py:1544
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Ви можете видалити старі ядра за допомогою команди «sudo apt autoremove» а "
"також додати COMPRESS=xz до файла /etc/initramfs-tools/initramfs.conf для "
"зменшення розмірів initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Очистіть кошик та видаліть тимчасові пакунки попередніх інсталяцій за "
"допомогою команди «sudo apt-get clean»."

#: ../DistUpgrade/DistUpgradeController.py:1552
msgid "Reboot to clean up files in /tmp."
msgstr "Для очищення файлів у  /tmp , необхідне перезавантаження."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1591
#: ../DistUpgrade/DistUpgradeController.py:2451
#: ../DistUpgrade/DistUpgradeController.py:2503
msgid "Calculating the changes"
msgstr "Проводиться аналіз змін"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1635
msgid "Do you want to start the upgrade?"
msgstr "Бажаєте почати оновлення системи?"

#: ../DistUpgrade/DistUpgradeController.py:1654
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""
"Оновлення захисту Livepatch для Ubuntu %s є недоступними. Якщо ви оновите "
"систему, Livepatch буде вимкнено."

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "Upgrade canceled"
msgstr "Оновлення скасовано"

#: ../DistUpgrade/DistUpgradeController.py:1727
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Оновлення буде скасовано та систему буде відновлено до попереднього стану. "
"Ви зможете продовжити оновлення пізніше."

#: ../DistUpgrade/DistUpgradeController.py:1733
#: ../DistUpgrade/DistUpgradeController.py:1878
msgid "Could not download the upgrades"
msgstr "Неможливо завантажити пакунки для оновлення системи"

#: ../DistUpgrade/DistUpgradeController.py:1734
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте ваше інтернет-з'єднання чи "
"носій, з якого здійснюється встановлення, та спробуйте знову. Всі "
"завантажені файли будуть збережені."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1827
#: ../DistUpgrade/DistUpgradeController.py:1866
#: ../DistUpgrade/DistUpgradeController.py:1983
msgid "Error during commit"
msgstr "Помилка при фіксації"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1829
#: ../DistUpgrade/DistUpgradeController.py:1868
#: ../DistUpgrade/DistUpgradeController.py:2032
msgid "Restoring original system state"
msgstr "Відновлення первісного стану системи"

#: ../DistUpgrade/DistUpgradeController.py:1830
#: ../DistUpgrade/DistUpgradeController.py:1845
#: ../DistUpgrade/DistUpgradeController.py:1869
msgid "Could not install the upgrades"
msgstr "Неможливо провести оновлення системи"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1835
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Оновлення було перервано. Ваша система може бути в нестабільному стані. "
"Зараз буде запущено процес відновлення (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1840
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Будь ласка, повідомте про цю помилку на сторінці за адресою "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug, "
"вклавши файли з /var/log/dist-upgrade/ до звіту про помилку.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1879
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте Ваше з'єднання з Інтернетом "
"чи носій для встановлення та спробуйте знову. "

#: ../DistUpgrade/DistUpgradeController.py:1907
msgid "Searching for obsolete software"
msgstr "Пошук застарілих програм"

#: ../DistUpgrade/DistUpgradeController.py:1971
msgid "Remove obsolete packages?"
msgstr "Видалити застарілі пакунки?"

#: ../DistUpgrade/DistUpgradeController.py:1972
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Залишити"

#: ../DistUpgrade/DistUpgradeController.py:1972
msgid "_Remove"
msgstr "Видалити"

#: ../DistUpgrade/DistUpgradeController.py:1984
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"При очищенні системи виникли проблеми. Прочитайте детальнішу інформацію "
"нижче. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:2070
msgid "Required depends is not installed"
msgstr "Необхідний пакет (залежність) не встановлений"

#: ../DistUpgrade/DistUpgradeController.py:2071
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Необхідний пакет (залежність) '%s' не встановлений. "

#. coherence check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:2304
#: ../DistUpgrade/DistUpgradeController.py:2399
msgid "Checking package manager"
msgstr "Перевірка програми управління пакунками"

#: ../DistUpgrade/DistUpgradeController.py:2310
#: ../DistUpgrade/DistUpgradeController.py:2316
msgid "Preparing the upgrade failed"
msgstr "Підготовка до оновлення не вдалася"

#: ../DistUpgrade/DistUpgradeController.py:2311
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Підготовка системи до оновлення закінчилася невдало. Почато процес "
"звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:2317
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Підготування системи до оновлення закінчилося невдачею. Щоб повідомити про "
"помилку, встановіть apport та виконайте команду 'apport-bug ubuntu-release-"
"upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:2334
#: ../DistUpgrade/DistUpgradeController.py:2344
msgid "Getting upgrade prerequisites failed"
msgstr "Помилка отримання необхідних файлів для оновлення"

#: ../DistUpgrade/DistUpgradeController.py:2335
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Системі не вдалося отримати попередні конфігурації для повного оновлення. "
"Зараз процес оновлення буде перервано і буде відновлено початковий стан "
"системи.\n"
"\n"
"Також почато процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:2345
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"Система не змогла отримати потрібні умови для оновлення. Оновлення буде "
"зараз перевано й відновиться початковий стан системи.\n"
"\n"
"Повідомити про помилку встановлення apport й потім виконати 'apport-bug "
"ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:2379
msgid "Updating repository information"
msgstr "Оновлення інформації про сховище"

#: ../DistUpgrade/DistUpgradeController.py:2428
msgid "Invalid package information"
msgstr "Невірна інформація про пакунок"

#: ../DistUpgrade/DistUpgradeController.py:2429
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Після оновлення інформації про пакунок не вдалося знайти важливий пакунок "
"'%s'. Причиною цього може бути відсутність офіційних дзеркал у вашому списку "
"джерел програмного забезпечення або тому, що дзеркало, яке ви "
"використовуєте, перевантажено. Див. поточний список налаштованих джерел "
"програмного забезпечення в /etc/apt/sources.list.\n"
"Якщо дзеркало перевантажено, можна знову спробувати оновитися згодом."

#: ../DistUpgrade/DistUpgradeController.py:2458
#: ../DistUpgrade/DistUpgradeController.py:2569
msgid "Fetching"
msgstr "Отримання"

#: ../DistUpgrade/DistUpgradeController.py:2465
#: ../DistUpgrade/DistUpgradeController.py:2488
#: ../DistUpgrade/DistUpgradeController.py:2518
#: ../DistUpgrade/DistUpgradeController.py:2573
msgid "Upgrading"
msgstr "Процес оновлення"

#: ../DistUpgrade/DistUpgradeController.py:2467
msgid "Upgrade infeasible"
msgstr "Оновлення неможливе"

#: ../DistUpgrade/DistUpgradeController.py:2468
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr "Неможливо виконати оновлення, під час оновлення були помилки."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2491
#: ../DistUpgrade/DistUpgradeController.py:2507
msgid "Upgrade incomplete"
msgstr "Оновлення неповне"

#: ../DistUpgrade/DistUpgradeController.py:2492
#: ../DistUpgrade/DistUpgradeController.py:2508
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"Оновлення частково завершено, але під час процесу оновлення були помилки."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2523
#: ../DistUpgrade/DistUpgradeController.py:2575
#: ../DistUpgrade/DistUpgradeController.py:2581
#: ../DistUpgrade/DistUpgradeController.py:2592
msgid "Upgrade complete"
msgstr "Оновлення завершено"

#: ../DistUpgrade/DistUpgradeController.py:2524
#: ../DistUpgrade/DistUpgradeController.py:2576
#: ../DistUpgrade/DistUpgradeController.py:2582
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Оновлення було завершено, але в його процесі виникли деякі помилки."

#: ../DistUpgrade/DistUpgradeController.py:2539
msgid "System upgrade is complete."
msgstr "Оновлення системи завершено."

#: ../DistUpgrade/DistUpgradeController.py:2593
msgid "The partial upgrade was completed."
msgstr "Часткове оновлення завершене."

#: ../DistUpgrade/DistUpgradeController.py:2614
msgid "Lock screen disabled"
msgstr "Екран блокування вимкнено"

#: ../DistUpgrade/DistUpgradeController.py:2615
msgid ""
"Your lock screen has been disabled and will remain disabled during the "
"upgrade."
msgstr ""
"Ваш екран блокування вимкнено і залишиться вимкненим під час оновлення."

#: ../DistUpgrade/DistUpgradeController.py:2623
msgid "Unable to disable lock screen"
msgstr "Не вдається вимкнути екран блокування"

#: ../DistUpgrade/DistUpgradeController.py:2624
msgid ""
"It is highly recommended that the lock screen be disabled during the upgrade "
"to prevent later issues. Please ensure your screen lock is disabled before "
"continuing."
msgstr ""
"Настійно рекомендується вимкнути екран блокування під час оновлення, щоб "
"запобігти подальшим проблемам. Перш ніж продовжити, переконайтеся, що "
"блокування екрана вимкнено."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Не вдалося знайти примітки випуску."

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Сервер може бути перенавантажений. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Не вдалося завантажити примітки випуску."

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherCore.py:77
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Будь ласка, перевірте ваше з'єднання з Інтернетом."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
msgid "Could not find the release announcement"
msgstr "Не вдалося знайти оголошення щодо випуску"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
msgid "The server may be overloaded."
msgstr "Можливо, сервер перевантажено."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:76
msgid "Could not download the release announcement"
msgstr "Не вдалося отримати оголошення щодо випуску"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:82
#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Продовжити [yN] "

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeFetcherCore.py:84
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "y"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:101
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "автентифікувати '%(file)s' замість '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:127
#, python-format
msgid "extracting '%s'"
msgstr "видобування '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:148
#: ../DistUpgrade/DistUpgradeFetcherCore.py:149
msgid "Could not run the upgrade tool"
msgstr "Неможливо провести апргрейд системи"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:150
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Найімовірнішою причиною може бути помилка в інструменті оновлення. Повідомте "
"про цю помилку за допомогою команди 'ubuntu-bug ubuntu-release-upgrader-"
"core'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Підпис інструменту оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Інструмент оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Не вдалося отримати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Не вдалося отримати оновлення. Це може бути викликано проблемою у мережі. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Помилка аутентифікації"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Автентифікація оновлення зазнала краху. Це може бути викликано проблемою з "
"мережею або сервером. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Не вдалося розпакувати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося розпакувати оновлення. Можливо, виникла проблема в мережі або на "
"сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Перевірка не вдала"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося перевірити наявність оновлень. Можливо, виникла проблема в мережі "
"або на сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Не вдається почати оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Це, зазвичай, виникає в системах з /tmp змонтованому з noexec. Перемонтуйте, "
"будь-ласка, без noexec та виконайте оновлення знов."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Повідомлення про помилку: '%s'."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Оновити"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
msgid "&Cancel"
msgstr "&Скасувати"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Відомості про релізе"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Оновлення"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Завантаження додаткових файлів пакунків..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s з %s при %sБ/с"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Файл %s з %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:127
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Вставте '%s' в привід '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Зміна носія"

#: ../DistUpgrade/DistUpgradeQuirks.py:242
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Ваше графічне обладнання не може бути повністю підтримуване в Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:244
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""
"Середовище робочого столу 'unity' підтримується вашим графічним апаратним "
"забезпеченням не повністю. Можливо, після оновлення швидкодія середовища "
"буде сповільнена. Радимо на разі залишити версію LTS. Щоб отримати докладні "
"відомості, див. "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Ви все ще "
"хочете продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:268
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Ваш графічний пристрій підтримується Ubuntu 12.04 LTS не повністю."

#: ../DistUpgrade/DistUpgradeQuirks.py:270
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Підтримку Вашого графічного адаптера Intel в Ubuntu 12.04 LTS обмежено, і Ви "
"можете стикнутися з проблемами після оновлення. Для отримання детальної "
"інформації відвідайте "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Ви дійсно бажаєте "
"продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:290
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Оновлення може уповільнити ефекти робочого столу, продуктивність роботи в "
"іграх та інших графічно складних програмах."

#: ../DistUpgrade/DistUpgradeQuirks.py:294
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Цей комп'ютер зараз використовує графічний драйвер AMD 'fglrx'. Нема "
"доступних версій драйверів, які б працювали з вашим обладнанням на Ubuntu "
"10.04 LTS.\n"
"\n"
"Чи хочете ви продовжувати?"

#: ../DistUpgrade/DistUpgradeQuirks.py:333
#: ../DistUpgrade/DistUpgradeQuirks.py:402
msgid "Sorry, no more upgrades for this system"
msgstr "Вибачте, цю систему вже не можна оновити"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""
"Подальших випусків Ubuntu для архітектури «i386» цієї системи не буде.\n"
"\n"
"Пакунки оновлень Ubuntu %s випускатимуть до %s."

#: ../DistUpgrade/DistUpgradeQuirks.py:349
msgid "No ARMv6 CPU"
msgstr "Немає ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:350
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ваша система використовує процесор ARM з архітектурою, старішою за ARMv6. "
"Всі пакети у karmic були скомпільовані з оптимізаціями, які вимагають ARMv6 "
"як мінімум. Оновлення вашої системи до нового випуску Ubuntu неможливе на "
"даному обладнанні."

#: ../DistUpgrade/DistUpgradeQuirks.py:370
msgid ""
"Sorry, this storage driver is not supported in kernels for newer releases"
msgstr ""
"Вибачте, підтримку драйвера до цього сховища даних не передбачено у ядрах "
"новіших випусків"

#: ../DistUpgrade/DistUpgradeQuirks.py:372
#, python-format
msgid ""
"There will not be any further Ubuntu releases that provide kernel support "
"for the aufs storage driver.\n"
"\n"
"Please ensure that none of your containers are using the aufs storage "
"driver, remove the directory %s and try again."
msgstr ""
"У подальших випусках Ubuntu у ядрі не буде передбачено підтримки драйвера "
"сховищ даних aufs.\n"
"\n"
"Будь ласка, переконайтеся, що у жодному із контейнерів не використано "
"драйвера сховища даних aufs, вилучіть каталог %s і повторіть спробу."

#: ../DistUpgrade/DistUpgradeQuirks.py:403
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's POWER8 "
"processor.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""
"Більше випусків Ubuntu для процесора POWER8 цієї системи не буде.\n"
"\n"
"Оновлення для Ubuntu %s триватимуть до %s."

#: ../DistUpgrade/DistUpgradeQuirks.py:421
msgid "No init available"
msgstr "Служба init недоступна"

#: ../DistUpgrade/DistUpgradeQuirks.py:422
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Схоже, що ваша система є віртуалізованим оточенням без служби init, "
"наприклад Linux-VServer. Ubuntu 10.04 LTS не може працювати в такому типі "
"оточення, ви повинні спочатку виправити конфігурацію своєї віртуальної "
"машини.\n"
"\n"
"Ви впевнені, що хочете продовжити?"

#: ../DistUpgrade/DistUpgradeQuirks.py:463
msgid "The Robot Operating System (ROS) is installed"
msgstr "Robot Operating System (ROS) встановлено"

#: ../DistUpgrade/DistUpgradeQuirks.py:464
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Схоже, ROS вже встановлено. Кожен випуск ROS дуже вимогливий до версій "
"Ubuntu, які він підтримує, тому оновлення Ubuntu можуть бути невдалими, якщо "
"ці вимоги не дотримані. Перш ніж продовжити, або видаліть ROS, або "
"переконайтеся, що встановлений випуск ROS підтримує версію Ubuntu, на яку ви "
"оновлюєтеся.\n"
"\n"
"Інформація про випуски ROS 1 вказана в REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"Інформація про випуски ROS 2 вказана в REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Ви дійсно хочете продовжити?"

#: ../DistUpgrade/DistUpgradeQuirks.py:657
msgid "Connection to the Snap Store failed"
msgstr "Не вдалося встановити з'єднання із Snap Store"

#: ../DistUpgrade/DistUpgradeQuirks.py:658
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""
"У вашій системі встановлено пакунок lxd, але не вдалося встановити з'єднання "
"із Snap Store. У поточному стані роботу lxd забезпечує snap. Оновлення "
"випуску системи не вдасться виконати, якщо не працюватиме snapd. Будь ласка, "
"переконайтеся, що комп'ютер з'єднано із інтернетом і оновіть параметри "
"брандмауера та проксі-сервера так, щоб можна було встановити зв'язок із "
"api.snapcraft.io. Якщо ви працюєте у промисловій мережі із брандмауером, "
"варто налаштувати проксі-сервер Snap Store."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "Connection to Snap Store failed"
msgstr "Помилка з'єднання з магазином Snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to "
"api.snapcraft.io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Ваша система не має підключення до Snap Store. Щоб отримати найкращу "
"процедуру оновлення, переконайтеся, що система може підключитися до "
"api.snapcraft.io. Продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:683
msgid "Outdated snapd package"
msgstr "Застарілий пакунок snapd"

#: ../DistUpgrade/DistUpgradeQuirks.py:684
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"У системі відсутній snapd останньої версії. Будь ласка, оновіть версію snapd "
"у системі, щоб мати покращену процедуру оновлення. Продовжити оновлення?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:709
msgid "Calculating snap size requirements"
msgstr "Обчислюємо вимоги щодо розміру для snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:740
msgid "Processing snap replacements"
msgstr "Обробляємо замінники snap"

#. TODO: This status should be updated, but the time of
#. this change to snap switch is post-translation freeze.
#: ../DistUpgrade/DistUpgradeQuirks.py:748
#, python-format
msgid "refreshing snap %s"
msgstr "оновлюємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:752
#, python-format
msgid "removing snap %s"
msgstr "вилучаємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:755
#, python-format
msgid "installing snap %s"
msgstr "встановлюємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:784
msgid "PAE not enabled"
msgstr "PAE не увімкнуто"

#: ../DistUpgrade/DistUpgradeQuirks.py:785
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"У вашій системі використовується процесор, у якому не увімкнено інструкцій "
"PAE. Користуватися Ubuntu у системах без PAE можна лише до версії Ubuntu "
"12.10. Оновлення до новішої версії Ubuntu вимагатиме вмикання PAE (якщо це "
"можливо). Див.: http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1081
msgid "universe component not enabled"
msgstr "компонент universe не увімкнено"

#: ../DistUpgrade/DistUpgradeQuirks.py:1082
#, python-format
msgid ""
"You have the package %s installed which is a python2 package. python2 has "
"been deprecated in Ubuntu 20.04 LTS and is now available from the universe "
"component of the Ubuntu archive. To ensure any existing third party code you "
"have installed continues to work the release upgrade process would install "
"%s for you. However, the universe component of the archive is not enabled on "
"this system. For the upgrade process to proceed you should either enable the "
"universe component in /etc/apt/sources.list or remove %s"
msgstr ""
"У вашій системі встановлено пакунок %s, який є пакунком python2. python2 "
"було визнано застарілим в Ubuntu 20.04 LTS. Тепер пакунки цієї версії "
"доступні з компоненту universe архіву Ubuntu. Щоб забезпечити продовження "
"роботи будь-якого наявного стороннього коду, процес оновлення випуску "
"встановить для вас %s. Втім, у цій системі не увімкнено компонент universe "
"архіву. Щоб процес оновлення змін продовжив роботу, вам слід або увімкнути "
"компонент universe в /etc/apt/sources.list або вилучити %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:1194
msgid "Checking for installed snaps"
msgstr "Перевірка на встановлені snap-пакунки"

#: ../DistUpgrade/DistUpgradeQuirks.py:1641
msgid "Cannot upgrade this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:1642
msgid ""
"Due to a bug in grub, this system cannot be safely upgraded at this time.\n"
"\n"
"See https://launchpad.net/bugs/2039172."
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Використовувати інтерфейс. В наявності: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ЗАСТАРІЛЕ* Цю опцію буде проігноровано"

#: ../DistUpgrade/DistUpgradeMain.py:68
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Виконати лише часткове оновлення (без перезапису sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid "Disable GNU screen support"
msgstr "Вимкнути підтримку GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Set datadir"
msgstr "Вказати каталог з даними"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Upgrade to the development release"
msgstr "Оновитися до випуску що розробляється"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:148
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Отримання завершено"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Отримання файлу %li з %li на швидкості %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:165
#: ../DistUpgrade/DistUpgradeViewGtk3.py:327
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Приблизно '%s' залишилось"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:168
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Отримання файлу %li з %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:200
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Застосування змін"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:226
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "проблеми з залежностями - залишається неналаштованим"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:231
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Не вдалося встановити '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:232
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Обновлення продовжиться, але пакет '%s' може бути в неробочому стані. Будь "
"ласка, надішліть звіт про помилку."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:249
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Замінити змінений користувачем конфігураційний файл\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:250
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ви втратите всі зроблені Вами у цьому конфігураційному файлі зміни якщо "
"заміните його новою версією."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:270
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "Команда 'diff' не знайдена"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:542
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Виникла невиправна помилка"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:543
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку (якщо ви ще це не зробили) та додайте "
"фали /var/log/dist-upgrade/main.log та /var/log/dist-upgrade/apt.log до "
"вашого звіту. Оновлення було перервано.\n"
"Ваш оригінальний sources.list було збережено в "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:573
msgid "Ctrl-c pressed"
msgstr "Натиснено Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:574
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Операція буде скасована, що може залишити систему в несправному стані. Ви "
"все ще хочете це зробити?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:713
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Для запобігання втрати інформації закрийте усі програми та документи."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:727
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Більше не підтримується Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:728
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Встановлення старої версії (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:729
#, python-format
msgid "Remove (%s)"
msgstr "Видалити (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:730
#, python-format
msgid "No longer needed (%s)"
msgstr "Більш не потрібен (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:731
#, python-format
msgid "Install (%s)"
msgstr "Встановлення (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:732
#, python-format
msgid "Upgrade (%s)"
msgstr "Оновлення (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:780
msgid "Cancel"
msgstr "Скасувати"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:781
#: ../DistUpgrade/DistUpgradeViewKDE.py:914
msgid "Continue"
msgstr "Далі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Показати відмінності >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Сховати відмінності"

#: ../DistUpgrade/DistUpgradeViewKDE.py:676
msgid "Error"
msgstr "Помилка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:728
msgid "Show Terminal >>>"
msgstr "Показати Термінал >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:731
msgid "<<< Hide Terminal"
msgstr "<<< Сховати Термінал"

#: ../DistUpgrade/DistUpgradeViewKDE.py:804
msgid "Information"
msgstr "Інформація"

#: ../DistUpgrade/DistUpgradeViewKDE.py:852
msgid "&Start Upgrade"
msgstr "&Почати оновлення"

#: ../DistUpgrade/DistUpgradeViewKDE.py:854
#: ../DistUpgrade/DistUpgradeViewKDE.py:898
#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Деталі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:876
#, python-format
msgid "No longer supported %s"
msgstr "Більш не підтримується %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:878
#, python-format
msgid "Remove %s"
msgstr "Видалити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:880
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Видалити (був автоматично інстальований) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:882
#, python-format
msgid "Install %s"
msgstr "Встановити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:884
#, python-format
msgid "Upgrade %s"
msgstr "Оновити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:931
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Необхідне перезавантаження"

#: ../DistUpgrade/DistUpgradeViewKDE.py:931
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Перезавантажте систему для завершення оновлення</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:934
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "Перезапустити зараз"

#: ../DistUpgrade/DistUpgradeViewKDE.py:935
msgid "&Close"
msgstr "&Закрити"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:952
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Скасувати запущене оновлення?</big></b>\n"
"\n"
"Система може потрапити у непридатний стан, якщо скасувати оновлення. Вам "
"наполегливо рекомендується продовжити оновлення."

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "Cancel Upgrade?"
msgstr "Скасувати оновлення?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li день"
msgstr[1] "%li дні"
msgstr[2] "%li днів"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li година"
msgstr[1] "%li години"
msgstr[2] "%li годин"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li хвилина"
msgstr[1] "%li хвилини"
msgstr[2] "%li хвилин"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li секунда"
msgstr[1] "%li секунди"
msgstr[2] "%li секунд"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 40Mbit connection and about %s with "
"a 5Mbit connection."
msgstr ""
"Це завантаження займе приблизно %s із з’єднанням 40 Мбіт і приблизно %s із "
"з’єднанням 5 Мбіт."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Це завантаження триватиме приблизно %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Підготовка до оновлення"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Отримання нових каналів програм"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Отримання нових пакунків"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Встановлення оновлень"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Очищення"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d установлений пакет більш не підтримується Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[1] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[2] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] ""
"Copy text   \t\r\n"
"%d пакет буде видалено."
msgstr[1] "%d пакети буде видалено."
msgstr[2] "%d пакетів буде видалено."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Буде встановлено %d новий пакет."
msgstr[1] "Буде встановлено %d нових пекети."
msgstr[2] "Буде встановлено %d нових пекетів."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Буде оновлено %d пакунок."
msgstr[1] "Буде оновлено %d пакунки."
msgstr[2] "Буде оновлено %d пакунків."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Вам треба завантажити всього %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Встановлення оновлень може зайняти декілька годин. Після завершення "
"завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Отримання та встановлення оновлень може зайняти декілька годин. Після "
"завершення завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "Видалення пакунків може зайняти декілька годин. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "Програмне забезпечення на цьому комп’ютері не потребує оновлення."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Немає оновлень для Вашої системи. Оновлення буде скасовано."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Необхідно перезавантажити систему"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Виконання оновлення завершено. Необхідно перезавантажити систему. "
"Перезавантажити зара?"

#: ../DistUpgrade/DistUpgradeView.py:434
msgid "Action required"
msgstr "Потрібна дія"

#: ../DistUpgrade/DistUpgradeView.py:435
msgid "Exit all other instances of Ubuntu WSL before continuing."
msgstr "Перш ніж продовжити, закрийте всі інші екземпляри Ubuntu WSL."

#: ../DistUpgrade/DistUpgradeView.py:436
msgid "Unsaved progress may otherwise be lost."
msgstr "Не збережені дані можуть бути втрачені."

#: ../DistUpgrade/DistUpgradeView.py:439
msgid "WSL restart required"
msgstr "Потрібно перезапустити WSL"

#: ../DistUpgrade/DistUpgradeView.py:440
msgid "Exit this instance of Ubuntu WSL."
msgstr "Вийти з цього екземпляра Ubuntu WSL."

#: ../DistUpgrade/DistUpgradeView.py:441
msgid "The upgrade will then be complete."
msgstr "Оновлення буде завершено."

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Виконано: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку та додайте файли  /var/log/dist-"
"upgrade/main.log та /var/log/dist-upgrade/apt.log до вашого звіту. Оновлення "
"було перервано.\n"
"Ваш оригінальний sources.list було збережено в "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Перервано"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Понижено версію:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Щоб продовжити натисніть [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Подробиці [d]"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Більше не підтримується: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Видалити: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Встановити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Оновити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Продовжити [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Потрібне перезавантаження для завершення оновлення.\n"
"Оберіть 'y' для перезавантаження."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"Завантаження файлу %(current)li з %(total)li із швидкістю %(speed)s/с"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Завантаження файлу %(current)li з %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Показувати прогрес для окремих файлів"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Скасувати оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Продовжити оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Відмінити оновлення системи?</big></b>\n"
"\n"
"Зауважте, що якщо Ви скасуєте оновлення, це може призвести до нестабільного "
"стану системи. Дуже рекомендується продовжити оновлення."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Почати оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Замінити"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Різниця між файлами"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "Повідомити про помилку"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "Пр_одовжити"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Почати оновлення?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Перезавантажте Вашу систему для завершення оновлення</big></b>\n"
"\n"
"Будь ласка, збережіть Вашу дані перед продовженням."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Оновлення дистрибутивів"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Оновлюємо Ubuntu до версії %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Встановлення нових каналів програм"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Перезавантаження системи"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Термінал"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Оновлення"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Доступна нова версія Ubuntu. Хочете оновити?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Не оновлювати"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Запитати пізніше"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Так, оновити зараз"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Ви відмовилось від оновлення Ubuntu до нової версії"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Можна виконати оновлення пізніше, відкривши Оновлювач програм і натиснувши "
"\"Оновити\"."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Виконати оновлення випуску"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Для оновлення Ubuntu, потрібна автентифікація."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Виконати часткове оновлення"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Для часткового оновлення, ви повинні пройти автентифікацію"

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Показати версію та вийти"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Каталог, який вміщує файли даних."

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Запустити вказаний інтерфейс оболонки"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Проводиться часткове оновлення"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Завантаження засобу оновлення випуску"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Якщо використовується найсвіжіший підтримуваний випуск, оновитися до "
"випуску, що перебуває у розробці"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:188
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Спробуйте оновитись до останньої версії за допомогою $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Запустити в особливому режимі оновлення.\n"
"Наразі підтримуються режим \"стільниця\" для регулярного оновлення "
"персональних робочих станцій та \"сервер\" для серверних систем."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Перевіряти доступність нової версії дистрибутива і повертати результат за "
"допомогою кода виходу."

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""
"Спробуйте оновлення із увімкненими сторонніми дзеркалами та сховищами, "
"замість спроби із закоментованими записами дзеркал і сховищ."

#: ../do-release-upgrade:114
msgid ""
"A comma-separated list of environment variables (e.g. "
"VAR1=VALUE1,VAR2=VALUE2) that should be set during the upgrade."
msgstr ""
"Розділений комами список змінних середовища (наприклад, VAR1=VALUE1, "
"VAR2=VALUE2), які потрібно встановити під час оновлення."

#: ../do-release-upgrade:126
msgid "The options --devel-release and --proposed are"
msgstr "Параметри --devel-release і --proposed"

#: ../do-release-upgrade:127
msgid "mutually exclusive. Please use only one of them."
msgstr ""
"не можна використовувати одночасно. Будь ласка, використовуйте лише один з "
"них."

#: ../do-release-upgrade:131
msgid "Checking for a new Ubuntu release"
msgstr "Перевіряється наявність нового випуску Ubuntu"

#: ../do-release-upgrade:143
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ваша версія Ubuntu вже не підтримується."

#: ../do-release-upgrade:144
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Для отримання інформації щодо оновлення, будь ласка, відвідайте:\n"
"%(url)s\n"

#: ../do-release-upgrade:159
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr "У /etc/update-manager/release-upgrades Prompt "

#: ../do-release-upgrade:160
msgid "is set to never so upgrading is not possible."
msgstr "встановлено на ніколи, тому оновлення неможливе."

#: ../do-release-upgrade:162
msgid "There is no development version of an LTS available."
msgstr "Версії, яка перебувала б у розробці, для LTS не існує."

#: ../do-release-upgrade:163
msgid "To upgrade to the latest non-LTS development release "
msgstr ""
"Щоб оновити систему до найсвіжішого випуску, який не є випуском LTS і "
"перебуває у розробці, "

#: ../do-release-upgrade:164
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""
"встановіть значення Prompt=normal у /etc/update-manager/release-upgrades."

#: ../do-release-upgrade:166
msgid "Upgrades to the development release are only "
msgstr "Оновлення до випуску, що перебуває у розробці, доступні "

#: ../do-release-upgrade:167
msgid "available from the latest supported release."
msgstr "лише для найсвіжішого підтримуваного випуску."

#: ../do-release-upgrade:169
msgid "No new release found."
msgstr "Нових випусків не знайдено."

#: ../do-release-upgrade:174
msgid "Release upgrade not possible right now"
msgstr "Оновлення реліза зараз неможливо."

#: ../do-release-upgrade:175
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Оновлення реліза не може бути виконано зараз, будь ласка, спробуйте пізніше. "
"Сервер повідомив: '%s'"

#: ../do-release-upgrade:181
#, c-format
msgid "New release '%s' available."
msgstr "Доступний новий реліз '%s' ."

#: ../do-release-upgrade:182
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Виконайте команду 'do-release-upgrade' для оновлення."

#: ../do-release-upgrade:199
msgid ""
"Please install all available updates for your release before upgrading."
msgstr ""
"Перед модернізацією, будь ласка, встановіть усі наявні оновлення для вашого "
"випуску."

#: ../do-release-upgrade:215
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"Ви не перезавантажилися після оновлення пакета, який цього потребує. Будь "
"ласка, перезавантажтесь перед модернізацією."

#: ../check_new_release_gtk.py:88
msgid "Software Updater"
msgstr "Оновлювач програм"

#: ../check_new_release_gtk.py:120
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Доступне оновлення Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:151
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ви відмовились від оновлення до Ubuntu %s"

#: ../check_new_release_gtk.py:184
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Перевірити можливість оновлення до останнього випуску, що перебуває в стані "
"розробки"

#: ../check_new_release_gtk.py:193
msgid "Add debug output"
msgstr "Додати результати відладки"
